<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>application</class>
 <widget class="QMainWindow" name="application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>application</string>
  </property>
  <widget class="QWidget" name="main">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="control_box">
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QGridLayout" name="control_lyt">
       <item row="0" column="0">
        <widget class="QLabel" name="min_lbl">
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="min">
         <property name="text">
          <string>-60</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="max_lbl">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="max">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="sequence_btn">
         <property name="text">
          <string>Sequence</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="pattern_box">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>Pattern</string>
      </property>
      <layout class="QGridLayout" name="pattern_lyt">
       <item row="0" column="0">
        <widget class="QWidget" name="plot" native="true">
         <layout class="QGridLayout" name="plot_lyt"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>min</tabstop>
  <tabstop>max</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
